UPGRADE-3.2.md

```diff
@@ -38,8 +38,36 @@ Form
 FrameworkBundle
 ---------------
 
-  * The service `serializer.mapping.cache.doctrine.apc` is deprecated. APCu should now
-    be automatically used when available.
+ * The service `serializer.mapping.cache.doctrine.apc` is deprecated. APCu should now
+   be automatically used when available.
+
+HttpKernel
+----------
+
+ * `DataCollector::varToString()` is deprecated and will be removed in Symfony
+   4.0. Use the `cloneVar()` method instead.
+
+   Before:
+
+   ```php
+   public function collect(Request $request, Response $response, \Exception $exception = null)
```

Comment:
I'm not sure these snippets are that useful, there is nothing special here, what about removing them (same below in UPGRADE-4.0.md)?